src/main/java/io/confluent/connect/elasticsearch/ElasticsearchClient.java

```diff
@@ -0,0 +1,97 @@
+/**
+ * Copyright 2018 Confluent Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ **/
+
+package io.confluent.connect.elasticsearch;
+
+import com.google.gson.JsonObject;
+import io.confluent.connect.elasticsearch.bulk.BulkRequest;
+import io.confluent.connect.elasticsearch.bulk.BulkResponse;
+import org.apache.kafka.connect.data.Schema;
+
+import java.io.IOException;
+import java.util.List;
+import java.util.Set;
+
+public interface ElasticsearchClient {
+
+  enum Version {
+    ONE, TWO, FIVE, SIX
```

Comment:
When imported statically (see below) it's not immediately obvious that this numbering is referring to ES versions. 

Not sure what the code style says about number in variable, but maybe something like: 
`ES_V1, ES_V2, ES_V5, ES_V6`
or
`ES1, ES2, ES5, ES6` wouldn't be too terrible. 

Not crazy about any of the alternatives. Up to you.

Reply:
Ok, I don't have a strong preference so I'll change to ES_V1, etc.